common/settings/settings.ts

```diff
@@ -149,7 +150,8 @@ const subtitleSettingsKeysObject: { [key in keyof SubtitleSettings]: boolean } =
     subtitleCustomStyles: true,
     subtitleBlur: true,
     imageBasedSubtitleScaleFactor: true,
-    subtitlePositionOffset: true,
+    bottomSubtitlePositionOffset: true,
```

Comment:
Use `subtitlePositionOffset` for `bottomSubtitlePositionOffset` and add a comment saying it's the bottom offset. The naming is less clear but without a system to migrate the settings structure it's safer to preserve older names, since the extension and app share settings.